### PR TITLE
chore: add text area spec

### PR DESCRIPTION
**Pull request by chrisdholt**
# Description
This PR adds a new spec for a text area component.

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: specs/text-area/text-area.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+*Screenshots below are of the basic appearance of the component and are not exhaustive.*
+
+| State | Image |
+| ----- | ----- |
+| default | ![](./images/text-area.png) |
+| focus | ![](./images/text-area-focus.png)
+| placeholder | ![](./images/text-area-placeholder.png)
+
+*Slot Names*
+- default - the content of the text area. Unlike input where content is a value, the content of a text area is predefined children.
```
Comment (janechu) [inline specs/text-area/text-area.md]: Could you re-word to "Unlike an input which stores a value and has no content, text area can have content which is treated as the value. This content must always be text."

File: specs/text-area/text-area.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+- label - the label content
```
Comment (janechu) [inline specs/text-area/text-area.md]: Should the images show a one with a label?

Comment (chrisdholt) [reply specs/text-area/text-area.md]: We can - I don't technically have an image for that or filled...so I didn't include those. Happy to though.

File: specs/text-area/text-area.md (hunk 1 above)
Comment (chrisdholt) [reply specs/text-area/text-area.md]: sure.

File: specs/text-area/text-area.md (hunk 2 above)
Comment (chrisdholt) [reply specs/text-area/text-area.md]: added

File: specs/text-area/text-area.md (hunk 1 above)
Comment (chrisdholt) [reply specs/text-area/text-area.md]: updated!

Comment (chrisdholt) [reply specs/text-area/text-area.md]: I changed this slightly as it seems we don't need slot projection to pass the children here

**Comment by chrisdholt** [comment]
> Approving this, but I'm wondering about another approach. What if we took our existing text field WC and gave it a `multiline` attribute and when that was true we used a text area instead of an input? (or we could add a new type value of textarea) Would that reduce duplication? Seems there's a lot of similarities.

I agree that there is some duplication and similarities in terms of attr's, but I think this might end up making the control a bit complex. In the text field scenario, I think a goal is to support easy addition of things like glyphs, or perhaps even a button (TBD). I'm not sure how that would work with a textarea from a design standpoint - though it could be explored.

@EisenbergEffect Perhaps this can be something to track and gather convergence on as part of Open-UI?

**Pull request merged**